### ISSUE TITLE
Bug: Deleting a Page: The input does not contain any JSON tokens

### ISSUE THREAD

**Issue by ADefWebserver** [oqtane/oqtane.framework#124]
To reproduce:
1) As the Host, create a new page
2) Try to delete the page

You will get the following error:

```
System.Text.Json.JsonException
  HResult=0x80131500
  Message=The input does not contain any JSON tokens. Expected the input to start with a valid JSON token, when isFinalBlock is true. Path: $ | LineNumber: 0 | BytePositionInLine: 0.
  Source=System.Text.Json
  StackTrace:
   at System.Text.Json.ThrowHelper.ReThrowWithPath(ReadStack& readStack, JsonReaderException ex)
   at System.Text.Json.JsonSerializer.ReadCore(JsonSerializerOptions options, Utf8JsonReader& reader, ReadStack& readStack)
   at System.Text.Json.JsonSerializer.ReadCore(Type returnType, JsonSerializerOptions options, Utf8JsonReader& reader)
   at System.Text.Json.JsonSerializer.ParseCore(String json, Type returnType, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.Deserialize[TValue](String json, JsonSerializerOptions options)
   at Oqtane.Repository.PermissionRepository.DecodePermissions(String PermissionStrings, Int32 SiteId, String EntityName, Int32 EntityId) in C:\Siliqon\oqtane\sbwalker.oqtane.framework\Oqtane.Server\Repository\PermissionRepository.cs:line 163
   at Oqtane.Repository.PermissionRepository.UpdatePermissions(Int32 SiteId, String EntityName, Int32 EntityId, String Permissions) in C:\Siliqon\oqtane\sbwalker.oqtane.framework\Oqtane.Server\Repository\PermissionRepository.cs:line 71
   at Oqtane.Repository.PageRepository.DeletePage(Int32 PageId) in C:\Siliqon\oqtane\sbwalker.oqtane.framework\Oqtane.Server\Repository\PageRepository.cs:line 65
   at Oqtane.Controllers.PageController.Delete(Int32 id) in C:\Siliqon\oqtane\sbwalker.oqtane.framework\Oqtane.Server\Controllers\PageController.cs:line 89
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.<>c__DisplayClass33_0.<WrapVoidMethod>b__0(Object target, Object[] parameters)
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.VoidResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()

Inner Exception 1:
JsonReaderException: The input does not contain any JSON tokens. Expected the input to start with a valid JSON token, when isFinalBlock is true. LineNumber: 0 | BytePositionInLine: 0.
```

**Comment by fileman** [comment]
I've solved in #122 
In PageRepository Delete method has UpdatePermissions instead of DeletePermissions

**Comment by sbwalker** [comment]
Closing as @fileman resolved this issue in his PR
